docs/content/integrations/dbt/reference.mdx

```diff
@@ -123,6 +123,81 @@ In production, `DAGSTER_DBT_PARSE_PROJECT_ON_LOAD` should be unset so that your
 
 ---
 
+## Using config with `@dbt_assets`
+
+Like for a standard \[software-defined asset], `@dbt_assets` can use a config system to enable [run configuration](/concepts/configuration/config-schema). This allows to provide parameters to jobs at the time they're executed.
+
+In the context of dbt, this can be useful if you want to run commands or flags for specific use cases. For instance, you may want to run [dbt seed](https://docs.getdbt.com/reference/commands/seed) or add [the --full-refresh flag](https://docs.getdbt.com/reference/resource-configs/full_refresh) to your dbt commands in some cases. Using a config system, the `@dbt_assets` object can be easily modified to support this use case.
+
+```python startafter=start_config_dbt_assets endbefore=end_config_dbt_assets file=/integrations/dbt/dbt.py dedent=4
+from pathlib import Path
+
+from dagster import AssetExecutionContext, Config
+from dagster_dbt import (
+    DbtCliResource,
+    DbtProject,
+    dbt_assets,
+)
+
+my_project = DbtProject(
+    project_dir=Path(__file__).joinpath("..", "./path/to/my-dbt-project").resolve(),
+)
+
+class MyDbtConfig(Config):
+    full_refresh: bool
+    seed: bool
```

Comment:
We should just focus on `full_refresh` as configuration, since it's the most common use case for Dagster config & `dagster-dbt`.

Adding `seed` doesn't really make sense here since `dbt build` materializes your dbt seeds as well.

Reply:
Good call